data-es2016plus.js

```diff
@@ -37,6 +37,7 @@ exports.tests = [
           safaritp: true,
           webkit: true,
           duktape20: true,
+          duktape21: true,
```

Comment:
Right, but how does one know X is newer than Y in the same family? I guess it's based on ordering in environments.json, i.e. same family but later in the file is newer?